site-src/index.md

```diff
@@ -1,3 +1,83 @@
 # Introduction
 
-TODO
\ No newline at end of file
+Gateway API Inference Extension is an official Kubernetes project focused on
+extending [Gateway API](https://gateway-api.sigs.k8s.io/) with inference
+specific routing extensions.
+
+The overall resource model focuses on 2 new inference-focused
+[personas](/concepts/roles-and-personas) and corresponding resources that
+they are expected to manage:
+
+<!-- Source: https://docs.google.com/presentation/d/11HEYCgFi-aya7FS91JvAfllHiIlvfgcp7qpi_Azjk4E/edit#slide=id.g292839eca6d_1_0 -->
+<img src="/images/resource-model.png" alt="Gateway API Inference Extension Resource Model" class="center" width="550" />
+
+## API Resources
+
+### InferencePool
+
+### InferenceModel
+
+## Composable Layers
+
+This project aims to develop an ecosystem of implementations that are fully
+compatible with each other. There are three distinct layers of components that
+are relevant to this project:
+
+### Gateway API Implementations
+
+Gateway API has [more than 25
+implementations](https://gateway-api.sigs.k8s.io/implementations/). As this
+pattern stabilizes, we expect a wide set of these implementations to support
+this project.
+
+### Endpoint Selection Extension
+
+As part of this project, we're building an initial reference extension that is
+focused on routing to LoRA workloads. Over time, we hope to see a wide variety
```

Comment:
I prefer not to give the impression that this is centered on LoRA, in reality LoRA is just one of multiple criteria for selection.